### PR TITLE
Upgrade to angular v19 & nx 20

### PR DESCRIPTION
**Pull request by hoebbelsB**
Upgraded nx & angular dependencies.

I think it doesn't create a breaking change, right? Otherwise I can rephrase the commit message to include a breaking change

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const { getJestProjects } = require('@nrwl/jest');
+const { getJestProjects } = require('@nx/jest');
 
 export default {
   projects: getJestProjects(),
```
Comment (ChristopherPHolder) [inline jest.config.ts:4]: Nx version change should have change this to use `getJestProjectsAsync`

File: package.json [hunk 2]
```diff
@@ -28,78 +27,79 @@
...
     "@types/node": "^18.16.9",
-    "@typescript-eslint/eslint-plugin": "7.9.0",
-    "@typescript-eslint/parser": "7.9.0",
-    "@typescript-eslint/utils": "^8.0.0-alpha.28",
+    "@typescript-eslint/eslint-plugin": "7.18.0",
+    "@typescript-eslint/parser": "7.18.0",
+    "@typescript-eslint/utils": "7.18.0",
     "compression": "^1.7.4",
     "cypress": "13.13.0",
     "domino": "^2.1.6",
```
Comment (ChristopherPHolder) [inline package.json:87]: I guess domino is not a dependency anymore since the angular fork

Comment (ChristopherPHolder) [reply package.json:87]: Does not need to be removed in this MR just noticed as i reviewed the changes

**Review by ChristopherPHolder — APPROVED** [review]
I guess if you dont remove the standalone flag its not a breaking change and we dont have to upgrade the peerDependencies in the package.json

File: jest.config.ts (hunk 1 above)
Comment (hoebbelsB) [reply jest.config.ts:4]: yeah, it didn't :D

**Pull request merged**